CRM/Financial/BAO/FinancialType.php

```diff
@@ -246,6 +246,42 @@ public static function permissionedFinancialTypes(&$permissions, $descriptions)
   }
 
   /**
+   * CRM-21026
+   * Wrapper aroung getAvaliableFinancialTypes to get all including disabled FinancialTypes
+   * @param array $financialTypes
+   *   (reference ) an array of financial types
```

Comment:
passing as reference is done badly in Civi code so I discourage it. In this case it is also returned & not required & in fact I think the parameter should not be pass in at all but rather initiated as an empty array within the function